frontend/src/lib/types.ts

```diff
@@ -28,6 +28,38 @@ export const signUpSchema = z
     message: 'Confirm Password do not match',
     path: ['confirmPassword'],
   });
+const isValidImageLink = (value: string) => {
+  const imageLinkRegex = /\.(jpg|jpeg|png|webp)$/i;
```

Comment:
Do we actually need a function? Can't we inline using the .validate option which zod provides directly

Reply:
.validate is when we are giving some url to test whether that url meets the defined schema or not @krishnaacharyaa

Reply:
@Ameerjafar okay if we can't inline then fine, but i guess inline/arrow function can be written